llpc/context/llpcCompiler.cpp

```diff
@@ -1418,7 +1418,11 @@ Result Compiler::BuildGraphicsPipeline(const GraphicsPipelineBuildInfo *pipeline
   BinaryData elfBin = {};
 
   SmallVector<const PipelineShaderInfo *, ShaderStageGfxCount> shaderInfo = {
-      &pipelineInfo->vs, &pipelineInfo->tcs, &pipelineInfo->tes, &pipelineInfo->gs, &pipelineInfo->fs,
+    &pipelineInfo->vs,
+    &pipelineInfo->tcs,
+    &pipelineInfo->tes,
+    &pipelineInfo->gs,
+    &pipelineInfo->fs,
```

Comment:
Why did the formatting change? If you want to keep it vertical, please surround this vector with a clang-format-off/on comments

Reply:
internal code is the format of:
&pipelineInfo->vs,
#if XXXX
some code here
&pipelineInfo->tcs,

so need to keep it vertical.

Reply:
Use clang-format on / off: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code.
This PR should pass the code formatting check